src/devices/Bmp180/Bmp180.cs

```diff
@@ -81,21 +81,32 @@ public double ReadPressure()
             int p = (B7 < 0x80000000) ? (int)((B7 * 2) / B4) : (int)((B7 / B4) * 2);
             X1 = (((p * p) / 65536 ) * 3038) / 65536;
             
-            return p + ( ((((p * p) / 65536 ) * 3038) / 65536) + ((-7357 * p) / 65536) + 3791) / 8;
+            return Pressure.FromPa(p + ( ((((p * p) / 65536 ) * 3038) / 65536) + ((-7357 * p) / 65536) + 3791) / 8);
         }
 
         /// <summary>
-        ///  Calculates the altitude in meters from the specified sea-level pressure(in hPa).
+        ///  Calculates the altitude in meters from the specified sea-level pressure.
         /// </summary>
         /// <param name="seaLevelPressure"> 
-        ///  Sea-level pressure in hPa
+        ///  Sea-level pressure
         /// </param>
         /// <returns>
         ///  Height in meters from the sensor
         /// </returns>
-        public double ReadAltitude(double seaLevelPressure = 101325.0)
+        public double ReadAltitude(Pressure seaLevelPressure)

```

Comment:
`Pressure seaLevelPressure = Pressure.MeanSeaLevel`?

Reply:
> `Pressure seaLevelPressure = Pressure.MeanSeaLevel`?

You can't use defaults because Pressure.MeanSeaLevel is not a constant value.